### PR TITLE
GAL-4078: show number of posts in community page tab on web

### PR DESCRIPTION
**Pull request by SouravInsights**
### Summary of Changes

show number of posts in community page tab on web

### Demo or Before and After

**Before:**
CleanShot 2023-08-22 at 20 32 00@2x

**After:**
CleanShot 2023-08-22 at 20 25 14@2x
CleanShot 2023-08-22 at 20 34 37@2x

### Edge Cases

List any common edge cases that you have considered and tested.

### Testing Steps

Provide steps on how the reviewer can test the changes.

### Checklist

Please make sure to review and check all of the following:

- [x] The changes have been tested and all tests pass.
- [x] WEB: The changes have been tested on various desktop screen sizes to ensure responsiveness.
- [ ] MOBILE APP: The changes have been tested on both light and dark modes.

### CODE REVIEW

File: apps/web/src/scenes/CommunityPage/CommunityPageTabs.tsx [hunk 1]
```diff
@@ -1,18 +1,52 @@
...
+    for (const edge of community.posts?.edges ?? []) {
+      if (edge?.node?.__typename === 'Post' && edge.node) {
+        events.push(edge.node);
+      }
+    }
+
+    return events;
+  }, [community.posts?.edges]);
+
+  const totalPosts = feedData?.length;
```
Comment (kaitoo1) [inline apps/web/src/scenes/CommunityPage/CommunityPageTabs.tsx:48]: we should use pageInfo.total from the connection instead , that will show the accurate total unaffected by pagination limits

Comment (SouravInsights) [reply apps/web/src/scenes/CommunityPage/CommunityPageTabs.tsx:48]: oh yeah.. right right!

**Comment by jakzaizzat** [comment]
Some edge cases that might need to test is:

Try to post something from the community page and see whether the post count is changed

**Comment by kaitoo1** [comment]
great call @jakzaizzat ! 

I just pushed changes to make sure the post count updates when creating and deleting a post.


https://github.com/gallery-so/gallery/assets/80802871/191d1921-318c-46b4-8a0b-1461f5bb5b3d



fyi @SouravInsights , there was also a warning for the unused edges field so I pushed a change to handle that
CleanShot 2023-08-24 at 13 25 33

**Review by kaitoo1 — APPROVED** [review]

**Pull request merged**